client/src/entrypoints/admin/userbar.js

```diff
@@ -261,6 +264,65 @@ class Userbar extends HTMLElement {
     function clickOutside() {
       hideUserbar();
     }
+
+    document.addEventListener('DOMContentLoaded', async () => {
+      await this.initialiseAxe();
+    });
```

Comment:
I think we should also only do this if the userbar has the "accessibility" menu item configured?